pay-id-api-spec/pay-id.v1.yml

```diff
@@ -99,6 +99,15 @@ paths:
             schema:
               $ref: '#/components/schemas/SignatureWrapper_Compliance'
       operationId: post-path-invoice
+      parameters:
+        - schema:
+            # Typed as a string to get around swagger-codegen issues.
+            # TODO(keefertaylor): Fix after https://github.com/swagger-api/swagger-codegen/issues/10105 is resolved
+            type: string
```

Comment:
If you want to be really specific, you could add `format: uuid` to this payload.